tests/backup/backup_test.go

```diff
@@ -549,6 +549,170 @@ var _ = Describe("{BasicBackupCreation}", func() {
 	})
 })
 
+var _ = Describe("{BasicSelectiveRestore}", func() {
+	var (
+		appList           []string
+		backupName        string
+		contexts          []*scheduler.Context
+		preRuleNameList   []string
+		postRuleNameList  []string
+		appContexts       []*scheduler.Context
+		bkpNamespaces     []string
+		clusterUid        string
+		clusterStatus     api.ClusterInfo_StatusInfo_Status
+		restoreName       string
+		cloudCredName     string
+		cloudCredUID      string
+		backupLocationUID string
+		bkpLocationName   string
+		numDeployments    int
+		providers         []string
+	)
+	backupLocationMap := make(map[string]string)
+	labelSelectors := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	selectedBkpNamespaces := make([]string, 0)
+	namespaceMapping := make(map[string]string)
+	JustBeforeEach(func() {
+		appList = Inst().AppList
+
+		if len(appList) == 1 {
+			numDeployments = 6 // Execute 'BasicBackupCreation' test case if there is only one app in the appList as this test case is irrelevant in such a scenario.
+		} else {
+			numDeployments = Inst().GlobalScaleFactor
+		}
```

Comment:
Do you mean to say if there are 2 apps provided, you will only deploy 2 namespaces?

Reply:
To clarify, I assumed that the default number of apps in the app-list would be one, and having more apps in the list would signify that the test command is specific to that particular test case. As you mentioned, even if there are two apps in the app-list, it only creates two namespaces. Should I set the default minimum number of deployments to six or another value?

Reply:
Set the minimum deployments to 6. Each app in the app-list will get deployed 6 times.